includes/class-bitcoin-donation-post-type.php

```diff
@@ -0,0 +1,277 @@
+<?php
```

Comment:
This whole file should be renamed since it's related to the shoutouts.

Or use it for all post types like the shoutouts and votings.